packages/components/pages/f-contact-preferences/src/store/_tests/contactPreferences.module.test.js

```diff
@@ -0,0 +1,149 @@
+import contactPreferencesModule from '../contactPreferences.module';
+import {
+    UPDATE_PREFERENCE,
+    UPDATE_PREFERENCES
+} from '../../constants';
+import {
+    contactPreferencesViewModel,
+    token,
+    contactPreferencesGetResponse,
+    contactPreferencesUpdateModel
+} from '../../../test-utils/setup';
+
+describe('ContactPreferences Store', () => {
+    let httpClientMock;
+    let getPreferencesMock;
+    let postPreferencesMock;
+
+    beforeEach(() => {
+        // Arrange
+        getPreferencesMock = jest.fn(() => contactPreferencesGetResponse);
+        postPreferencesMock = jest.fn(() => {});
```

Comment:
If you want it to return an object:
```suggestion
        postPreferencesMock = jest.fn(() => ({}));
```

Or if you want it to do nothing, you can just do this:
```suggestion
        postPreferencesMock = jest.fn();
```